app/test/Close.test.js

```diff
@@ -58,7 +58,7 @@ contract('Balance Redirect Presale, close() functionality', ([anyone, appManager
         expect(totalRaised).to.equal(contribution)
 
         // reserve
-        const tokensForReserve = web3.toBigNumber(totalRaised).mul(PPM + mintingForBeneficiaryPct).div(PPM).mul(RESERVE_RATIOS[0]).div(PPM)
+        const tokensForReserve = web3.toBigNumber(totalRaised).mul(PPM).div(PPM - mintingForBeneficiaryPct).mul(RESERVE_RATIOS[0]).div(PPM)
```

Comment:
This test is actually not very helpful as you are just writing the math in a different programming language. I am pretty sure you didn't catch the error because of this. Please add a test with real numbers.

Reply:
No, I didn't catch the error because I didn't consider it an error. I would have used the same numbers given by the formula I was using.
Actually, although I don't agree with this comment, I'm going to change, but I'm just going to use that formula to produce those "real numbers", so we'll have them hardcoded ones which in my opinion is worse.